### DISCUSSION TITLE
Setting breakpoint problem - is there a way to make nvim-dap ignore case in path?

### DISCUSSION THREAD

**Discussion by pavel-savchenko-deel** [mfussenegger/nvim-dap#1356] [Q&A]
I've come across an issue with setting a breakpoint, and dap logs says `Breakpoint unverified` with a correct path seen as "not found".

Googling brought me to this solution: https://www.reddit.com/r/neovim/comments/16aoa5u/neovimnvimdapgo_debugger_doesnt_trigger/

And indeed, restarting neovim after switching to the "correct" casing (containing upper an case letter) has solved it.

Seems that nvim-dap fails when I start neovim with the lower case variant, as it's doing a case sensitive search. I think we need to be looking up the path while disregarding case, at least on macOS (and possibly Windows)

**Comment by mfussenegger** [comment]
> Seems that nvim-dap fails when I start neovim with the lower case variant, as it's doing a case sensitive search. I think we need to be looking up the path while disregarding case

The debug adapter is resolving the breakpoints, so this should probably be done in the adapter.

If paths are case-sensitive or not depends on the operating system and its filesystem, and for the client (nvim-dap) it is as far as I can tell never safe to normalize the paths to lowercase or something like this.

You'd have to report this upstream with the debug adapter
